.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: rust
 cache: cargo
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libgoogle-perftools-dev
```

Comment:
if you don't run bench why do you need this ?

Reply:
Because it needs it at the linking of the executable. I still want to build it to be sure that I don’t break something in `benchmark.rs`